### PR TITLE
Add timing information option

### PR DESCRIPTION
**Pull request by alexr-bq**
### Summary

Log timing information of various connector operations, to help with troubleshooting and improving performance.

### Description

Adds an option to the connector, defaulting to true for now, to log information about how long different operations take:
- Export
- Partitioning
- Read
- Write 
- Config

### Related Issue

https://github.com/vertica/spark-connector/issues/315

### Additional Reviewers

@jonathanl-bq 
@jeremyp-bq 
@Aryex

### CODE REVIEW

**Review by Aryex — APPROVED** [review]
lgtm

**Comment by Aryex** [comment]
I think CI is failing because you are missing the license info in Timer class.

File: connector/src/main/scala/com/vertica/spark/datasource/core/VerticaDistributedFilesystemReadPipe.scala [hunk 1]
```diff
@@ -326,10 +337,14 @@ class VerticaDistributedFilesystemReadPipe(
   var partition : Option[VerticaDistributedFilesystemPartition] = None
   var fileIdx = 0
 
+
+  var t0 = System.currentTimeMillis();
```
Comment (jeremyp-bq) [inline connector/src/main/scala/com/vertica/spark/datasource/core/VerticaDistributedFilesystemReadPipe.scala:341]: Can we use the timer class here as well for consistency (I see that was done in the write class)?

Comment (alexr-bq) [reply connector/src/main/scala/com/vertica/spark/datasource/core/VerticaDistributedFilesystemReadPipe.scala:341]: Oops, missed one. Will do.

File: README.md [hunk 2]
```diff
@@ -108,46 +108,47 @@ There are a few ways to solve this:
...
+| time_operations                                | Boolean                                             | Whether parts of the connector operation should be timed. Timing information will be logged                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       | No                                             | true                        |
```
Comment (jeremyp-bq) [inline README.md:151]: Default is `false`, not `true`.

**Review by jeremyp-bq — APPROVED** [review]

**Pull request merged**